project/src/components/sort/sort.tsx

```diff
@@ -0,0 +1,63 @@
+import { Dispatch, useState } from 'react';
+import { connect, ConnectedProps } from 'react-redux';
+import { SortType } from '../../const';
+import { changeSortType } from '../../store/actions';
+import { State } from '../../types/state';
+import { Actions } from '../../types/action';
+
+const mapStateToProps = ({currentSortType}: State) => ({
+  currentSortType,
+});
+
+const mapDispatchToProps = (dispatch: Dispatch<Actions>) => ({
+  onSortTypeChange(sortType: SortType) {
+    dispatch(changeSortType(sortType));
+  },
+});
+
+const connector = connect(mapStateToProps, mapDispatchToProps);
+
+type PropsFromRedux = ConnectedProps<typeof connector>;
+
+function Sort({currentSortType, onSortTypeChange}: PropsFromRedux): JSX.Element {
+  const [dropdownOpened, setDropdownOpened] = useState<boolean>(false);
+  const sortTypes = Object.values(SortType);
+
+  const onSortTypeClick = (sortType: SortType): void => {
```

Comment:
```suggestion
  const handleSortTypeClick = (sortType: SortType): void => {
```
локальные колбеки названые через `handle`